controllers/tlspolicy_status.go

```diff
@@ -87,6 +88,18 @@ func (r *TLSPolicyReconciler) calculateStatus(ctx context.Context, tlsPolicy *v1
 	return newStatus
 }
 
+func (r *TLSPolicyReconciler) acceptedCondition(tlsPolicy *v1alpha1.TLSPolicy, specErr error) *metav1.Condition {
+	ok, err := kuadrantgatewayapi.IsCertManagerInstalled(r.RestMapper)
+	if err != nil {
+		return kuadrant.AcceptedCondition(tlsPolicy, err)
+	}
+	if !ok {
+		return kuadrant.AcceptedCondition(tlsPolicy, errors.New("cert-manager APIs not available"))
+	}
+
+	return kuadrant.AcceptedCondition(tlsPolicy, specErr)
+}
+
 func (r *TLSPolicyReconciler) enforcedCondition(ctx context.Context, tlsPolicy *v1alpha1.TLSPolicy, targetNetworkObject client.Object) *metav1.Condition {
```

Comment:
Is this relevant now since https://github.com/Kuadrant/kuadrant-operator/pull/715. The controller no longer gets started if the cert manager CRDs are missing so should never get here.

Reply:
yeah, reverting the changes done for the status reporting as they are useless. Pending tasks for a good reporting captured in https://github.com/Kuadrant/kuadrant-operator/issues/730

Reply:
done